src/main/java/am/rockstars/enums/ProductCategory.java

```diff
@@ -0,0 +1,5 @@
+package am.rockstars.enums;
+
+public enum ProductCategory {
+    MEN, WOMEN, GADGETS, ACCESSORIES
```

Comment:
we need to use low letters as value, you can see an implementation 
https://github.com/f985/aaa/blob/941735fdf8b805bc483f887936219e94bd18694f/src/main/java/am/rockstars/enums/HeaderType.java#L8